adafruit_circuitplayground/express.py

```diff
@@ -184,6 +191,30 @@ def red_led(self):
     def red_led(self, value):
         self._led.value = value
 
+    def play_tone(self, frequency, duration):
+        """ Produce a tone using the speaker.
+
```

Comment:
Please document the two arguments frequency and duration using the param style here: https://github.com/adafruit/Adafruit_CircuitPython_DotStar/blob/master/adafruit_dotstar.py#L39